packages/client/src/ui/ChessMain.tsx

```diff
@@ -197,6 +198,7 @@ const Game = () => {
           )
         )}
       </div>
+      <PreLoadAssets />
```

Comment:
Maybe it could be written that hooks don't need components

Reply:
> Maybe it could be written that hooks don't need components

@aLIEzsss4  For later development, there will a page for loading assets. Component would have something show up.